Cargo.toml

```diff
@@ -53,3 +53,10 @@ members = [
     "otl-normalizer",
     "fontc_crater",
 ]
+
+# TEMPORARY
```

Comment:
Please release fontations and remove this

Reply:
there are several other changes to read-fonts and write-fonts in fontations and I am unsure about bumping minor vs patch or whether there are other pending stuff that waits to be merged before release. Maybe @cmyr or @dfrg can take care of that?

Reply:
done